### PR TITLE
OSDOCS#11845:MTO Operator Upgrate from TP to GA

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s):
4.16+

Issue:
[OSDOCS-11845](https://issues.redhat.com/browse/OSDOCS-11845)

Link to docs preview:
- [Installing the Multiarch Tuning Operator by using the web console](https://84587--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.html#multi-architecture-installing-using-web-console_multiarch-tuning-operator)
- [Installing the Multiarch Tuning Operator by using the CLI](https://84587--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.html#multi-architecture-installing-using-cli_multiarch-tuning-operator)
- [Uinstalling the Multiarch Tuning Operator by using the CLI](https://84587--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/configuring-multi-arch-compute-machines/multiarch-tuning-operator.html#multi-architecture-uninstalling-using-cli_multiarch-tuning-operator)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sr1kar99** [comment]
@aleskandro 
Could you please review this PR?
Thanks!

File: modules/multi-arch-installing-using-cli.adoc [hunk 1]
```diff
@@ -92,7 +92,7 @@ $ oc get csv -n openshift-multiarch-tuning-operator
 [source,terminal]
 ----
 NAME                               DISPLAY                     VERSION   REPLACES   PHASE
-multiarch-tuning-operator.v0.9.0   Multiarch Tuning Operator   0.9.0                Succeeded
+multiarch-tuning-operator.v1.0.0   Multiarch Tuning Operator   1.0.0                Succeeded
```
Comment (lwan-wanglin) [inline modules/multi-arch-installing-using-cli.adoc:95]:
```suggestion
NAME                               DISPLAY                     VERSION   REPLACES                           PHASE
multiarch-tuning-operator.v1.0.0   Multiarch Tuning Operator   1.0.0     multiarch-tuning-operator.v0.9.0   Succeeded
```

File: modules/multi-arch-installing-using-cli.adoc [hunk 2]
```diff
@@ -122,5 +122,5 @@ $ oc get subscription -n openshift-multiarch-tuning-operator
 [source,terminal]
 ----
 NAME                        PACKAGE                     SOURCE                              CHANNEL
-multiarch-tuning-operator   multiarch-tuning-operator   multiarch-tuning-operator-catalog   tech-preview
+multiarch-tuning-operator   multiarch-tuning-operator   multiarch-tuning-operator-catalog   stable
```
Comment (lwan-wanglin) [inline modules/multi-arch-installing-using-cli.adoc:125]:
```suggestion
openshift-multiarch-tuning-operator   multiarch-tuning-operator   redhat-operators   stable
```

**Comment by lwan-wanglin** [comment]
others looks good to me, thanks @sr1kar99

File: modules/multi-arch-installing-using-cli.adoc (hunk 1 above)
Comment (sr1kar99) [reply modules/multi-arch-installing-using-cli.adoc]: Updated as per the suggestion.

File: modules/multi-arch-installing-using-cli.adoc (hunk 2 above)
Comment (sr1kar99) [reply modules/multi-arch-installing-using-cli.adoc]: Updated as per the suggestion.

**Review by skopacz1 — APPROVED** [review]
LGTM!

**Pull request merged**